students/uss_tomasz/lesson_05_lists/5_1/comma_code.py

```diff
@@ -0,0 +1,21 @@
+def list_to_string(list_from_user):
```

Comment:
in fact this function is should not care if the list is from user or not, so the name `list_from_user` is not the best choice